include/model.php

```diff
@@ -819,20 +823,20 @@ protected function get_languages_from_taxonomies() {
 		// Restore the right order after the first `array_reverse()`.
 		$terms_by_slug = array_reverse( $terms_by_slug );
 
-		$languages = array();
-
-		foreach ( $terms_by_slug as $lang_terms ) {
-			/**
-			 * @var (
-			 *     array{
-			 *         language: WP_Term,
-			 *         term_language: WP_Term
-			 *     }&
-			 *     array<non-empty-string, WP_Term>
-			 * ) $lang_terms
-			 */
-			$languages[] = PLL_Language_Factory::get_from_terms( $lang_terms );
-		}
+		/**
+		 * @var (
+		 *     array{
+		 *         string: array{
+		 *             language: WP_Term,
+		 *             term_language: WP_Term
+		 *         }&array<non-empty-string, WP_Term>
+		 *     }
+		 * ) $terms_by_slug
+		 */
```

Comment:
I know it's not related to this PR but I'd like to disuss this with you and @Screenfeed